tests/Instances.hs

```diff
@@ -136,6 +137,9 @@ instance (ApproxEq a) => ApproxEq [a] where
 instance Arbitrary Version where
     arbitrary = makeVersion . fmap getNonNegative <$> resize 4 (listOf1 arbitrary)
 
+instance Arbitrary a => Arbitrary (NonEmpty a) where
```

Comment:
I restricted `QuickCheck` to `<2.8.3` because of this.
I'll make a PR to `QuickCheck` to include this and `Version` instance (though different)